source/constructs/package.json

```diff
@@ -25,6 +26,7 @@
     "@types/node": "^18.8.2",
     "aws-cdk": "^2.61.1",
     "aws-cdk-lib": "^2.61.1",
+    "child_process": "^1.0.2",

```

Comment:
I noticed the [child_process](https://www.npmjs.com/package/child_process) package hasn't been touched in 7 years and accord to npm `This package name is not currently in use, but was formerly occupied by another package. To avoid malicious use, npm is hanging on to the package name, but loosely, and we'll probably give it to you if you want it.`.  I would be hesitant to include this in the solution. Can this be possibly replaced with something else, such as: https://nodejs.org/api/child_process.html#child_processexecsynccommand-options